tests/qxmppatmmanager/tst_qxmppatmmanager.cpp

```diff
@@ -60,15 +60,14 @@ private slots:
 
     QXmppClient m_client;
     QXmppLogger m_logger;
+    QXmppAtmTrustMemoryStorage m_trustStorage;
     QXmppAtmManager *m_manager;
-    QXmppAtmTrustMemoryStorage *m_trustStorage;
     QXmppCarbonManager *m_carbonManager;
 };
 
 void tst_QXmppAtmManager::initTestCase()
 {
-    m_trustStorage = new QXmppAtmTrustMemoryStorage;
-    m_manager = new QXmppAtmManager(m_trustStorage);
+    m_manager = new QXmppAtmManager(&m_trustStorage);
```

Comment:
Since we're changing this anyway, does it need to be heap-allocated? That is, why not have
```cpp
QXmppAtmManager m_manager { &m_trustStorage };
```
initializer in the class definition?

Reply:
Thanks, I changed it accordingly.